1.MLP_Binary_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,627 @@
+#!/bin/env/python
+from datetime import date
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+import seaborn as sns
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from sklearn.metrics import (
+    accuracy_score,
+    auc,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    precision_score,
+    recall_score,
+    roc_auc_score,
+    roc_curve,
+)
+from sklearn.model_selection import train_test_split
+
+from .parameters import Parameters
+
+params = Parameters()
```

Comment:
this is really elegant, I like!!

However, currently as written, there is no way to update params other than editing the class in `parameters.py` (it's also possible I'm missing something!)